client.go

```diff
@@ -289,6 +289,7 @@ func (c *HelmClient) install(ctx context.Context, spec *ChartSpec) (*release.Rel
 					Getters:          c.Providers,
 					RepositoryConfig: c.Settings.RepositoryConfig,
 					RepositoryCache:  c.Settings.RepositoryCache,
+					Out:              os.Stdout,
```

Comment:
Hey @guitarlum, first of all thanks for your contribution!

Rather than specifying `os.Stdout` at this point, i'd like to suggest expanding the [`helmclient.Options`](https://github.com/mittwald/go-helm-client/blob/master/types.go#L31) and the [`helmclient.HelmClient`](https://github.com/mittwald/go-helm-client/blob/master/types.go#L49) itself by adding a more generic `io.Writer` field (as suggested in https://github.com/mittwald/go-helm-client/issues/64). Users could then specify a custom `io.Writer` implementation via `Options` when creating new clients through [`helmclient.newClient`](https://github.com/mittwald/go-helm-client/blob/master/client.go#L74).  This might prove as a more versatile approach on the issue by giving users the freedom to decide what to do with `downloader.Output`. `os.Stdout` should only be set as a (configurable) default.

Reply:
Heya @elenz97 , 

thanks a lot for the feedback. 
I adjusted the parts according to your comment. Not sure how to write a test checking the private attribute of an implementation in go though. If you have any suggestions, please let me know.

Reply:
Hey @guitarlum, thanks for the update.

I think it would be sufficient to add a general usage example to our [code examples](https://github.com/mittwald/go-helm-client/blob/master/client_test.go#L10) in this case. If you could add the above, the PR is ready to go!

Reply:
Sorry for the late reply @elenz97 
I adjusted the code example; hope that looks fine to you.